### ISSUE TITLE
Legacy blocks can't be edited in the customizer

### ISSUE THREAD

**Issue by draganescu** [WordPress/gutenberg#25154]
**Describe the bug**
There is no way to publish block updates in the widget editor embedded in the customizer

**To reproduce**
1. Be on a theme that supports widgets and have the Gutenberg 8.9 plugin set up
2. Have a plugin that provides widgets
3. Go to Appearance > Widgets
4. Add a Legacy Widget block
5. Select the widget provided by your plugin 
6. Set up and save the widgets
7. Go to Customizer > Widget areas
8. Notice you cannot edit legacy widgets's settings

**Expected behavior**
Legacy widget blocks should have the same Edit / Preview behavior in the Customizer like they do in the Widgets editor.

**Comment by draganescu** [comment]
Related #25153

**Comment by adamziel** [comment]
Blocked by https://github.com/WordPress/gutenberg/issues/25625

**Comment by noisysocks** [comment]
Duplicate of https://github.com/WordPress/gutenberg/issues/25157.
